docs/en/advanced/Namespace-Management.md

```diff
@@ -53,32 +66,148 @@ metadata will be loaded into Alluxio the first time they are accessed (e.g. when
 open a file). The data of the file is not loaded to Alluxio during this process. To load the data into
 Alluxio, one can read the data using `FileInStream` or use the `load` command of the Alluxio shell.
 
-## Unified Namespace
 
-Alluxio provides a mounting API that makes it possible to use Alluxio to access data across multiple
-data sources.
+# Mounting Under File System (UFS)
+Mounting in Alluxio works similarly to mounting a volume in a Linux file system.
+The mount command attaches a UFS to the file system tree in the Alluxio namespace. 
 
-![unified]({{ site.baseurl }}/img/screenshot_unified.png)
+## Root Mount Point
+The root mount point of the Alluxio namespace can be specified by adding the following line to the configuration file in `conf/alluxio-site.properties`.
+Below is an example of configuration having an HDFS path to be the root of Alluxio namespace.
 
-By default, Alluxio namespace is mounted onto the directory specified by the
-`alluxio.underfs.address` property of Alluxio configuration; this directory identifies the
-"primary storage" for Alluxio. In addition, users can use the mounting API to add new and remove
-existing data sources:
+```
+alluxio.underfs.address=hdfs://HDFS_HOSTNAME:9000
+``` 
+
+If the root mount point requires additional parameters such as aws credentials, it can be specified in the configuration file as well. 
 
-```java
-void mount(AlluxioURI alluxioPath, AlluxioURI ufsPath);
-void mount(AlluxioURI alluxioPath, AlluxioURI ufsPath, MountOptions options);
-void unmount(AlluxioURI path);
-void unmount(AlluxioURI path, UnmountOptions options);
+```
+aws.accessKeyId=<AWS_ACCESS_KEY_ID>
```

Comment:
these should also be prefixed with `alluxio.master.mount.table.root.option.`, right?

Reply:
fixed